tests/reg_tests/refs/adjoint_euler_scalar_jst_tut_wing.json

```diff
@@ -1,86 +1,87 @@
 {
-    "Eval Functions Sens:": {
+    "Adjoint States": 0.0,
+    "Eval Functions Sens 2:": {
```

Comment:
As I mention below, I don't think we should add this, ideally the only thing added to the `.json` files is the `Adjoint States` entry